modules/administration/pages/iss.adoc

```diff
@@ -0,0 +1,34 @@
+[[iss]]
+= Inter-Server Synchronization
+
+If you have more than one {productname} installation, you will probably want to ensure that they stay aligned on content and permissions.
+Inter-Server Synchronization (ISS) allows you to connect two or more {productname} Servers and keep them up-to-date.
+
+To set up ISS, you need to define one {productname} Server as a master, with the other as a slave.
+This is so that, where conflicting configurations exist, the system will know which configuration to prioritize.
```

Comment:
Wording?  i'd write: "If conflicting configurations exist, the system knows ..."